README.md

```diff
@@ -46,31 +36,26 @@ also be changed.
 
 * Update the template constants (install dates and deadlines) in config/template_values.yml.
 
-## Initialize the Database
```

Comment:
But please restore the bit about resetting the db, which we need to do every year to clean out the old data.